### PR TITLE
[ASM] Expander - Remediation Guidance - February

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues

[EXPANDR-1568](https://jira-hq.paloaltonetworks.local/browse/EXPANDR-1568)

## Description

Monthly update of RemediationGuidance list to add new policy types.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
@BigEasyJ Hi, Do we need to wait for ASM-side approver? Or Can Merge this pr?

**Review by daryakoval — APPROVED** [review]

**Comment by BigEasyJ** [comment]
Yes we will review @daryakoval. Please wait; @capanw will review.

**Review by capanw — APPROVED** [review]

**Pull request merged**
